concent-builder/configure.yml

```diff
@@ -58,9 +75,34 @@
             state:  present
             groups: docker
 
+        - name:    Add jenkins user to sudo group
+          user:
```

Comment:
I don't think giving jenkins sudo powers is a good idea.

Reply:
Ok, my mistake here. However, we need add jenkins to sudoers to specify no passwd when the playbooks run tasks as root user. Maybe we need create different user for running playbooks ?

Reply:
The playbook that installs and configures jenkins is expected to be run by a user (e.g. your or my user) who has sudo access. But this needs to be done only once.

The playbook that is started by jenkins and runs tests should not have or need sudo access at all. If it does (e.g. because it tries to access files of a different user (e.g. `builder`), we need to change this). For example the pull request that allows building from accounts other than `builder` can help with that.

Reply:
Is it fixed?

https://github.com/golemfactory/concent-deployment/pull/284#discussion_r246439542

Reply:
The problem is in the `install-repository.yml` playbook when we clone repositories to `/var/concent-data`, we still use the `builder` user.

Reply:
#191 was meant to fix that. In that case you can leave it as is in this pull request but please create an issue to take away Jenkins' sudo access as soon as #191 is resolved.